.github/workflows/lint.yml

```diff
@@ -0,0 +1,18 @@
+name: Lint code
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.10
```

Comment:
Thoughts on following pre-commit's example config and use setup-python without specifying a Python  version?

Reply:
Fine by me, I'd brought this over from other projects but we can switch to `actions/setup-python@v3` per their Readme.